### PR TITLE
implemented the distinction between a transaction and a contract invo…

### PR DESCRIPTION
**Pull request by developer0623**
…cation without parameters

### CODE REVIEW

File: app/components/Transaction/index.js [hunk 1]
```diff
@@ -167,21 +167,24 @@ const getStatus = (transaction, selectedAccountHash, t) => {
   }
 
   if (type === types.TRANSACTION && isSameLocation) {
-    if (transaction.parameters) {
+    if (
+      !transaction.parameters &&
+      (transaction.consumed_gas === 10100 || transaction.consumed_gas === 10260)
```
Comment (anonymoussprocket) [inline app/components/Transaction/index.js]: Please use named constants.

File: app/reduxContent/invoke/thunks.js [hunk 2]
```diff
@@ -95,6 +95,9 @@ export function invokeAddress(
       }
 
       const clearedOperationId = clearOperationId(res.operationGroupID);
+      const consumedGas = operationResult.consumed_gas
+        ? parseInt(operationResult.consumed_gas, 10)
```
Comment (anonymoussprocket) [inline app/reduxContent/invoke/thunks.js]: "10" can be assumed, no?

Comment (developer0623) [reply app/reduxContent/invoke/thunks.js]: parse

Comment (anonymoussprocket) [reply app/reduxContent/invoke/thunks.js]: I know what it is, wouldn't parseInt(n) for normal user entry be equivalent to parseInt(n, 10); I doubt people will be typing 0x7A, etc.

File: app/components/Transaction/index.js [hunk 3]
```diff
@@ -8,6 +8,10 @@ import TezosIcon from '../TezosIcon/';
 import { openLinkToBlockExplorer } from '../../utils/general';
 import * as types from '../../constants/TransactionTypes';
 import { READY } from '../../constants/StatusTypes';
+import {
+  REGULAR_CONSUMED_GAS,
```
Comment (anonymoussprocket) [inline app/components/Transaction/index.js]: consumed for what, origination, transaction? Also, what's regular? "implicit" accounts are tz1,2,3; originated are kt1.

Comment (umurb) [reply app/components/Transaction/index.js]: How about we call these

REG_TX_GAS_CONSUMPTION
EMPTY_OUT_TX_GAS_CONSUMPTION

Comment (anonymoussprocket) [reply app/components/Transaction/index.js]: ok

**Pull request merged**